File: year2020/month09/day0922/gebal_0922.py
add = lambda x, y: (x + y) ** 3
sub = lambda x, y: (x - y) ** 2

a, b = list(map(int, input().split(' ')))
print(f"({a} - {b}) ^ 2 = {sub(a, b)}")
print(f"({a} + {b}) ^ 3 = {add(a, b)}")

File: year2020/month09/day0922/gebal_0922.c
#include<stdio.h>
#include<math.h>
#define sub(x, y) pow((x - y), 2)
#define add(x, y) pow((x + y), 3)

int main() {
	int a, b;
	scanf("%d %d", &a, &b);
	printf("(%d - %d) ^ 2 = %d\n", a, b, (int)sub(a,b));
	printf("(%d + %d) ^ 3 = %d\n", a, b, (int)add(a, b));
}
